emperor/support_files/js/emperor.js

```diff
@@ -728,9 +728,10 @@ function colorChangedForTaxaSpheres(color){
 function colorChangedForTaxaLabels(color){
         // get the taxonomic assignments and append '_taxalabel' to
         // retrieve all the labels belonging to a sphere in the plot
-        for(var key in g_taxaPositions) {
+        _.each(_.range(g_taxaPositions.length),function(key){
+        //for(var key in g_taxaPositions) {
        	        $('#'+key+"_taxalabel").css('color', color);
```

Comment:
Can you put spaces between + and string/vars? I think it makes it more readable .